### PR TITLE
Removes RoyalGorge KS link from welcome page

### PR DESCRIPTION
**Pull request by philcampeau**
## Before clicking "Create"

- [x] Branch is derived from the latest `master`
- [x] Add the `pins` or `archive_alpha_games` label if this change will break existing games
- [x] Code passes linter with `docker compose exec rack rubocop -a`
- [x] Tests pass cleanly with `docker compose exec rack rake`

### Explanation of Change

The 18Royalgorge kickstarter is over. Removing the link to it from the welcome page.

### CODE REVIEW

File: assets/app/view/welcome.rb [hunk 1]
```diff
@@ -16,13 +16,11 @@ def render
 
     def render_notification
       message = <<~MESSAGE
-        <p><a href='https://www.kickstarter.com/projects/18wood/18royalgorge/'>18RoyalGorge is now on Kickstarter</a>.</p>
-
         <p><a href="https://github.com/tobymao/18xx/wiki/18RoyalGorge">18RoyalGorge</a> is now in beta.</p>
 
-        <p><a href="https://github.com/tobymao/18xx/wiki/1858%20Switzerland">1858 Switzerland</a> is in alpha.</p>
+         <p><a href="https://github.com/tobymao/18xx/wiki/1858%20Switzerland">1858 Switzerland</a> is in alpha.</p>
```
Comment (ollybh) [inline assets/app/view/welcome.rb:21]: An extra space has been added at the start of this line. Can you please remove this?

File: assets/app/view/welcome.rb [hunk 2]
```diff
@@ -16,13 +16,11 @@ def render
...
 
-        <p>Report bugs and make feature requests <a href='https://github.com/tobymao/18xx/issues'>on GitHub</a>.</p>
+         <p>Report bugs and make feature requests <a href='https://github.com/tobymao/18xx/issues'>on GitHub</a>.</p>
```
Comment (ollybh) [inline assets/app/view/welcome.rb:23]: And there's a space to remove here too.

**Review by ollybh — CHANGES_REQUESTED** [review]

**Comment by philcampeau** [comment]
I thought that was odd too, but rubocop insisted on it.

**Comment by ollybh** [comment]
> I thought that was odd too, but rubocop insisted on it.

I don't get any complaints from Rubocop when I delete those two spaces. Can you try again and confirm?

File: assets/app/view/welcome.rb (hunk 1 above)
Comment (philcampeau) [reply assets/app/view/welcome.rb:21]: done

File: assets/app/view/welcome.rb (hunk 2 above)
Comment (philcampeau) [reply assets/app/view/welcome.rb:23]: done

**Review by ollybh — APPROVED** [review]

**Pull request merged**